testing/snapshot.ts

```diff
@@ -204,7 +204,7 @@ export function serialize(actual: unknown): string {
     compact: false,
     iterableLimit: Infinity,
     strAbbreviateSize: Infinity,
-  }).replace(/\\n/g, "\n");
+  }).replace(/\\n/g, "\n").replace(/\\t/g, "\t");
```

Comment:
We added option (`escapeSequences`) for disabling the escape of this kind of chars in https://github.com/denoland/deno/pull/19337. Let's use that option instead.

Reply:
Updated to use `escapeSequences: false`